src/main/java/org/apache/sling/graphql/core/engine/DefaultQueryExecutor.java

```diff
@@ -157,6 +158,13 @@ public class DefaultQueryExecutor implements QueryExecutor {
                         " Change ONLY if you know exactly what you are doing."
         )
         int maxWhitespaceTokens() default 200000;
+
+        @AttributeDefinition(
+                name = "Maximum Field Count",
+                description = "The number of fields queried with an GraphQL request. This is a safety measure to avoid denial of service attacks." +
+                        " Change ONLY if you know exactly what you are doing."
+        )
+        int maxFieldCount() default 200000;
```

Comment:
@schulm, why does Sling need 2x what the library sets by default? Since this is configurable, I'd like to go with the default. WDYT?

Reply:
I've set it to 100k